src/worker.cpp

```diff
@@ -1563,4 +1563,103 @@ void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
         */
     }
 }
+
+void Worker::deleteFruVpd(const std::string& i_dbusObjPath)
+{
+    if (i_dbusObjPath.empty())
+    {
+        throw std::runtime_error("Given DBus object path is empty.");
```

Comment:
There is a try catch in the API, wrap complete API into that. No point letting this exception go to the caller.
Log PEL in the catch and then either re throw or do what is required.

Reply:
Done